* this is not really necessary (ncurses provides both), but it solves a
  problem on case-insensitive filesystems where curses.h equals Curses.h

--- CUnit/Sources/Curses/Curses.c
+++ CUnit/Sources/Curses/Curses.c
@@ -52,7 +52,7 @@
 #include <assert.h>
 #include <string.h>
 #include <math.h>
-#include <curses.h>
+#include <ncurses.h>
 
 #include "CUnit.h"
 #include "TestDB.h"
